tests/test-js/test_event_metric.js.tmpl

```diff
@@ -0,0 +1,29 @@
+import { createEventsServerEventLogger } from "./server_events.js";
```

Comment:
This file shouldn't be necessary I think?

Reply:
Right, I eventually used your template with a slight modification, committed this by accident.